handsontable/src/editors/autocompleteEditor/__tests__/autocompleteEditor.spec.js

```diff
@@ -1054,94 +1053,216 @@ describe('AutocompleteEditor', () => {
       expect(editor.find('tbody td:eq(5)').text()).toEqual('6');
     });
 
-    it('should display the dropdown above the editor, when there is not enough space below the cell AND' +
-       'there is more space above the cell', async() => {
+    it('should display the dropdown above the editor, when there is not enough space below (table has defined size)', async() => {
       handsontable({
-        data: Handsontable.helper.createSpreadsheetData(30, 30),
-        columns: [
-          {
-            editor: 'autocomplete',
-            source: choices
-          }, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}
-        ],
+        data: createEmptySpreadsheetData(30, 30),
+        editor: 'autocomplete',
+        source: choices,
         width: 400,
-        height: 400
+        height: getDefaultRowHeight() * 9,
       });
 
-      setDataAtCell(29, 0, '');
-      selectCell(29, 0);
+      mouseDoubleClick($(getCell(6, 0)));
 
-      mouseDoubleClick($(getCell(29, 0)));
+      await sleep(50);
 
-      await sleep(200);
+      const container = $(getActiveEditor().htContainer);
 
-      const autocompleteEditor = $('.autocompleteEditor');
+      expect(container.offset()).forThemes(({ classic, main }) => {
+        classic.toEqual({ top: 23, left: 0 });
+        main.toEqual({ top: 29, left: 0 });
```

Comment:
shouldn't `23` and `29` be constant shared between tests? shouldn't `getDefaultRowHeight` be used here ?

Reply:
Fixed aea878d054d4a36bb3e915f4bcfaf799976fe4af